src/runtime_src/core/common/api/xrt_device.cpp

```diff
@@ -20,8 +20,8 @@
 #define XCL_DRIVER_DLL_EXPORT  // exporting xrt_device.h
 #define XRT_CORE_COMMON_SOURCE // in same dll as core_common
 
-#include "core/include/experimental/xrt_device.h"
-#include "core/include/experimental/xrt_aie.h"
+#include "core/include/xrt/xrt_device.h"
+#include "core/include/xrt/xrt_aie.h"
```

Comment:
Do we expect the `experimental` files to be renamed or removed if they are already in `xrt`?

Reply:
At some point yes.   Ideally I would like to remove them immediately when promoting the APIs from experimental to mature, but that would create a hassle for DSV in their sprite tests.   It's probably one of those things that we will never get around to?